test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1190,26 +1190,26 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				By("checking if ContainerRuntimeConfig object already exists")
 				ctrcfg, err := getContainerRuntimeConfigFrom(context.TODO(), profile, mcp)
 				Expect(err).ToNot(HaveOccurred(), "failed to get ContainerRuntimeConfig from profile %q mcp %q", profile.Name, mcp.Name)
+				if ctrcfg == nil {
```

Comment:
Can we set the `When` to `Ordered` and add this code in `BeforeAll()` closure?

Reply:
addressed in the latest commit